sample_projects/LEQuad/proj_avr32/config/conf_platform.h

```diff
@@ -46,7 +46,7 @@
 extern "C" {
 #endif
 
-#define MAVLINK_SYS_ID 0
+#define MAVLINK_SYS_ID 2
```

Comment:
done in https://github.com/lis-epfl/MAVRIC_Library/pull/311/commits/154be9e70e2c85dfc062b9c418bdc1fd48ee1bdb